packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -143,4 +143,210 @@ class CupertinoIcons {
+
+  /// An arrow on a circular path with its end pointing at its start surrounded by a circle. This is icon is filled in.
+  ///
+  /// See also:
+  ///
+  ///  * [refresh_circled], which is similar, but not filled in.
+  ///  * [refresh], which is the arrow of this icon filled in without a circle.
+  static const IconData refresh_circled_solid = IconData(0xf49c, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// An arrow on a circular path with its end pointing at its start.
+  ///
+  /// See also:
+  ///
+  ///  * [refresh], which is an arrow of the same concept, but thicker and with a larger gap in between its end and start.
+  static const IconData refresh_thin = IconData(0xf49d, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// An arrow on a circular path with its end pointing at its start.
+  ///
+  /// See also:
+  ///
+  ///  * [refresh], which is similar, but rotated 45 degrees anti-clockwise and thinner.
+  ///  * [refresh_bold], which is similar, but rotated 45 degrees clockwise and thicker.
+  static const IconData refresh_thick = IconData(0xf3a8, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// An arrow on a circular path with its end pointing at its start.
+  ///
+  /// See also:
+  ///
+  ///  * [refresh_thick], which is similar, but rotated 45 degrees anti-clockwise and thinner.
+  ///  * [refresh], which is similar, but rotated 90 degrees anti-clockwise and much thinner.
+  static const IconData refresh_bold = IconData(0xf21c, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A cross of two diagonal lines from edge to edge crossing in an angle of 90 degrees, which is used for dismissal.
+  ///
+  /// See also:
+  ///
+  ///  * [clear_circled], which uses this cross as a blank space in a filled out circled.
+  ///  * [clear_thin], which uses a thinner cross and is the iOS 7 equivalent of this icon.
+  static const IconData clear = IconData(0xf2d7, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A cross of two diagonal lines from edge to edge crossing in an angle of 90 degrees, which is used for dismissal, used as a blank space in a circle.
+  ///
+  /// See also:
+  ///
+  ///  * [clear], which is equivalent to the cross of this icon without a circle.
+  ///  * [clear_thin_circled_solid], which is similar, but uses a thinner cross.
+  static const IconData clear_circled = IconData(0xf36e, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A cross of two diagonal lines from edge to edge crossing in an angle of 90 degrees, which is used for dismissal.
+  ///
+  /// See also:
+  ///
+  ///  * [clear_thin_circled], which consists of this cross and a circle surrounding it.
+  ///  * [clear], which uses a thicker cross and is the pre-iOS 7 equivalent of this icon.
+  static const IconData clear_thin = IconData(0xf404, fontFamily: iconFont, fontPackage: iconFontPackage);
```

Comment:
Following the same weight as above, these 3 should be clear, clear circled and clear circled solid and the 2 above are clear_thick and clear_thick_circled.